docs/Tutorials/CrossCompilationSetup/CrossCompilationSetupTutorial.md

```diff
@@ -0,0 +1,59 @@
+# F´ Cross-Compilation Setup Tutorial
+
+## Table of Contents
+
+* <a href="#1-introduction">1. Introduction</a>
+  * <a href="#prerequisites"> Prerequisites</a>
+* <a href="#2-installing-dependencies">2. Installing Dependencies</a>
+* <a href="#3-installing-the-toolchain">3. Installing the Toolchain</a>
+
+<a name="Introduction"></a>
+## 1. Introduction
+
+In this section, we will learn how to install all the dependencies required for cross-compiling for different architectures.
+This tutorial will use the Raspberry Pi ARM x64 as an example. In order to fully benefit from this tutorial, the user should acquire a Raspberry Pi.
+
+<a name="Prerequisites"></a>
+### Prerequisites
+
+To run through this tutorial, you must have a computer that meets the following basic requirements.
+
+1. Computer running Windows 10, Mac OS X, or Ubuntu
+2. Administrator access
+3. 5GB of free disk space, 8 GB of RAM
+4. Knowledge of the command line for your operating system (Bash, Powershell, Zsh, etc).
+
+<a name="Installing Dependencies"></a>
+## 2. Installing Dependencies
+
+Choose the operating system you are using to install F Prime:
+
+- [Windows 10/11 WSL](./Windows.md)
+- [Mac OS X](./Mac.md)
```

Comment:
You link to Mac.md here, but I don't see that in our list of files.

Reply:
Added file